app/assets/javascripts/user.js

```diff
@@ -178,6 +220,42 @@ $(document).ready(function () {
     }
   }
 
+  function updateHomeLocation() {
+    const lat = $("#home_lat").val().trim();
+    const lon = $("#home_lon").val().trim();
+    if (!lat || !lon) {
+      return;
+    }
+
+    const geocodeUrl = `${OSM.NOMINATIM_URL}reverse?format=json&lat=${lat}&lon=${lon}`;
+
+    if (locationInput.request) locationInput.request.abort();
+    locationInput.request = $.getJSON(geocodeUrl, function (data) {
```

Comment:
blocked by content security policy

Reply:
Was updated to get information from the local call instead of directly accessing Nominatim.